docs/mastering-plone-5/instructions_plone5/index.md

```diff
@@ -0,0 +1,382 @@
+Starting server in PID 25230.
+Serving on http://0.0.0.0:8080
+```
+
+````{note}
+In rare cases when you are using macOS with an UTF-8 character set starting Plone might fail with the following error:
+
+```text
+ValueError: unknown locale: UTF-8
+```
+````
+
+In that case you have to put the localized keyboard and language settings in the .bash_profile
+of the vagrant user to your locale (like `en_US.UTF-8` or `de_DE.UTF-8`)
+
+```shell
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+```
+
+Now the Zope instance we're using is running.
+You can stop the running instance anytime using {kbd}`ctrl + c`.
+
+If it doesn't, don't worry, your shell isn't blocked.
+
+Type {kbd}`reset` (even if you can't see the prompt) and press RETURN, and it should become visible again.
+
+If you point your local browser at <http://localhost:8080> you see that Plone is running in Vagrant.
+
+This works because VirtualBox forwards the port 8080 from the guest system (the vagrant Ubuntu) to the host system (your normal operating system).
+
+There is no Plone site yet - we will create one in chapter 6.
+
+The Buildout for this Plone is in a shared folder.
+This means we run it in the vagrant box from {file}`/vagrant/buildout` but we can also access it in our own operating system and use our favorite editor.
+
+You will find the directory {file}`buildout` in the directory {file}`training` that you created in the beginning
+next to {file}`Vagrantfile` and {file}`manifests`.
+
+```{note}
+The database and the python packages are not accessible in your own system since large files cannot make use of symlinks in shared folders.
+The database lies in `/home/vagrant/var`, the python packages are in `/home/vagrant/packages`.
+```
+
+If you have any problems or questions please mail us at <mailto:team@starzel.de> or create a ticket at <https://github.com/plone/training/issues>.
+
+(plone5-instructions-vagrant-does-label)=
+
+### What Vagrant does
+
+Installation is done automatically by vagrant and puppet.
+If you want to know which steps are actually done please see the chapter {doc}`what_vagrant_does`.
+
+(plone5-instructions-vagrant-care-handling-label)=
+
+```{note}
+**Vagrant Care and Handling**
+
+Keep in mind the following recommendations for using your Vagrant VirtualBoxes:
+
+- Use the {command}`vagrant suspend` or {command}`vagrant halt` commands to put the VirtualBox to "sleep" or to "power it off" before attempting to start another Plone instance anywhere else on your machine, if it uses the same port.  That's because vagrant "reserves" port 8080, and even if you stopped Plone in vagrant, that port is still in use by the guest OS.
+- If you are done with a vagrant box, and want to delete it, always remember to run {command}`vagrant destroy` on it before actually deleting the directory containing it.  Otherwise you'll leave its "ghost" in the list of boxes managed by vagrant and possibly taking up disk space on your machine.
```

Comment:
```suggestion
- If you are done with a vagrant box, and want to delete it, always remember to run {command}`vagrant destroy` on it before actually deleting the directory containing it.  Otherwise you'll leave its "ghost" in the list of boxes managed by Vagrant and possibly taking up disk space on your machine.
```